### PR TITLE
append_withcustomflags

### PR DESCRIPTION
**Pull request by Subbireddy-Kuluru**
Mime append with custom flags

### CODE REVIEW

File: lib/Connection.js [hunk 1]
```diff
@@ -323,7 +323,9 @@ Connection.prototype.append = function(data, options, cb) {
       options.flags = [options.flags];
     if (options.flags.length > 0) {
       for (var i = 0, len = options.flags.length; i < len; ++i) {
-        if (options.flags[i][0] !== '\\')
+        //if (options.flags[i][0] !== '\\')
```
Comment (mscdex) [inline lib/Connection.js]: This commented code should be removed.

Comment (subbiwy) [inline lib/Connection.js]: OK. Now all commented code is cleaned and comitted.

**Comment by mscdex** [comment]
LGTM. Can you squash the two commits?

**Comment by Subbireddy-Kuluru** [comment]
I have done 'git Squash'. 

Unfortunately don't know git concepts properly and followed git help for Squashing, I am assuming the done proper git Squash'. If you required I will do one more pull to request and will commit the cleanup code.

**Comment by mscdex** [comment]
Landed in 98200e0b3f8. Thanks.

**Pull request closed without merging**